app/backends/service-metrics.ts

```diff
@@ -28,7 +28,7 @@ export async function getReadyWorkItemCountForServiceID(
   try {
     let workItemCount;
     await db.transaction(async (tx) => {
-      workItemCount = await workItemCountByServiceIDAndStatus(tx, serviceID, [WorkItemStatus.READY]);
+      workItemCount = await workItemCountByServiceIDAndStatus(tx, serviceID, [WorkItemStatus.READY, WorkItemStatus.RUNNING]);
```

Comment:
We should add a couple simple tests which have work items in various states and then verify the function returns the correct count. We may even be able to reuse existing tests with work items and just add a couple of additional asserts.

Reply:
Done.